src/main/java/com/liferay/faces/util/cache/Cache.java

```diff
@@ -62,12 +55,12 @@
 	 * @param  key    The key that the value is mapped to.
 	 * @param  value  The value to be cached.
 	 */
-	public V putIfAbsent(K key, V value);
+	public V putValueIfAbsent(K key, V value);
```

Comment:
@ngriffin7a, I think that naming a method `putValue` but having the behavior be `putIfAbsent` will confuse the maintenance dev. Not everyone will read the docs, but everyone reads the code.